pom.xml

```diff
@@ -606,6 +606,12 @@
         <groupId>${akka.group}</groupId>
         <artifactId>akka-remote_${scala.binary.version}</artifactId>
         <version>${akka.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>io.netty</groupId>
```

Comment:
Yeah, so this exclude is likely to break Akka remote now that the Netty 3.x classes are completely missing from the classpath.